### DISCUSSION TITLE
1-Minute or 5-Minute Temperature Data

### DISCUSSION THREAD

**Discussion by thecondor123** [weather-gov/api#447] [Ideas]
I know 1-minute and 5-minute temperature data is recorded at ASOS stations. Is it possible we could request either of those for the past hour or day?

**Comment by scadergit** [comment]
You can request previous observations by using the timestamp. If you don't know the timestamps, the /observations endpoint will list them for you.

**Reply by thecondor123** [reply]
Thanks, but for the current day it only provides hourly data. I was hoping for access to the more frequent updates.

**Reply by StephenClouse** [reply]
We do not currently receive the 1- or 5-minute ASOS feeds. Sorry about that.
